libsignal-service/src/content.rs

```diff
@@ -2,17 +2,16 @@ pub use crate::{
     proto::{
         attachment_pointer::Flags as AttachmentPointerFlags,
         data_message::Flags as DataMessageFlags, data_message::Reaction,
-        group_context::Type as GroupType, sync_message, AttachmentPointer,
-        CallMessage, DataMessage, GroupContext, ReceiptMessage, SyncMessage,
-        TypingMessage,
+        group_context::Type as GroupType, AttachmentPointer, CallMessage,
+        DataMessage, ReceiptMessage, SyncMessage, TypingMessage,
```

Comment:
The structs in the [`sync_message` module](`https://rubdos.gitlab.io/whisperfish/doc/libsignal_service/proto/sync_message/index.html`) are necessary in client implementation. That's why I added that to the `pub use`.

```sh
whisperfish
❯ rg sync_message
src/worker/client.rs
590:                use libsignal_service::content::sync_message;
593:                    sent: Some(sync_message::Sent {
```